x-pack/plugins/saved_objects_tagging/server/routes/tags/create_tag.ts

```diff
@@ -10,9 +10,13 @@ import type { TagsPluginRouter } from '../../types';
 import { TagValidationError } from '../../services/tags';
 
 export const registerCreateTagRoute = (router: TagsPluginRouter) => {
+  // I should mimic the versioned router API here, but let's just duplicate the routes instead
+  // so that TS is happy.
+
+  // v3 - color
   router.post(
     {
-      path: '/api/saved_objects_tagging/tags/create',
+      path: '/api/saved_objects_tagging/v3/tags/create',
       validate: {
         body: schema.object({
           name: schema.string(),
```

Comment:
Even if we don't have versioned router yet, this is an example of how we would support HTTP API versioning